src/sentry/incidents/models.py

```diff
@@ -417,6 +418,29 @@ class Meta:
         unique_together = (("alert_rule", "project"),)
 
 
+@region_silo_only_model
+class AlertRuleProject(Model):
+    """
+    Specify a project for the AlertRule
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+
+    alert_rule = FlexibleForeignKey("sentry.AlertRule", db_index=False)
+    project = FlexibleForeignKey("sentry.Project", db_constraint=False)
```

Comment:
I don't think we need to remove the constraint here, I'm not sure why we did it on the other table